lib/Models/TerriaJsonCatalogFunction.js

```diff
@@ -374,18 +416,24 @@ TerriaJsonCatalogFunction.prototype.invoke = function() {
 
     queryParameters[parameter.id] = parameter.formatForService();
   });
-
   const uri = new URI(this.url).addQuery(queryParameters);
   const proxiedUrl = proxyCatalogItemUrl(this, uri.toString(), "1d");
-
-  const promise = loadJson(proxiedUrl).then(json => {
+  const promise = Resource.fetchXHR({
+      url: proxiedUrl,
+      responseType: "text",
+      headers: {
+        Accept: "application/json,*/*;q=0.01"
+      },
+      returnType: "XHRJSONHEADERS"
+    }).then(all => this._handleHttp202(all)).then(json => {
```

Comment:
I'm not sure we should default to handling a 202 response like this. I think we might be better leaving this functionality "off" by default and having it selectively enabled in the constructor or something? That way consumers of this API can choose what they want to happen, and there isn't unexpected "magic" happening behind the scenes. @Jason-DNetto thoughts?

Reply:
isn't off by default returning a response with status code HTTP 200: OK?
sometimes protocols/servers return HTTP status code 200 with an XML document that has a status element with 202 in it. 
when i wrote this i had intended for consumers to control what happens when they assign acceptedUrl in the init file, and have been treating acceptedUrl === undefined as an error, but maybe this can be the off by default switch.